Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -1071,6 +978,84 @@ def fetch_incidents(client):
     demisto.incidents(incidents)
 
 
+def get_table_command(client, args):
```

Comment:
its preferred to use args.get with a default instead of direct access

Reply:
done